src/test/lib/logical_query_plan/except_node_test.cpp

```diff
@@ -27,9 +27,7 @@ class ExceptNodeTest : public BaseTest {
 
   std::shared_ptr<MockNode> _mock_node1, _mock_node2, _mock_node3;
   std::shared_ptr<ExceptNode> _except_node;
-  std::shared_ptr<LQPColumnExpression> _a;
-  std::shared_ptr<LQPColumnExpression> _b;
-  std::shared_ptr<LQPColumnExpression> _c;
+  std::shared_ptr<LQPColumnExpression> _a, _b, _c;
```

Comment:
We usually avoid that.

Reply:
I know we dow, but we also use this heavily in tests to not have 20 lines of expression declarations. So I guess we need to come up with a rule for that. TBH, I prefer https://github.com/hyrise/hyrise/blob/f7ad570cc30208c71593c0d7f609e34c0f60decb/src/test/lib/logical_query_plan/aggregate_node_test.cpp#L38 over https://github.com/hyrise/hyrise/blob/f7ad570cc30208c71593c0d7f609e34c0f60decb/src/test/lib/logical_query_plan/join_node_test.cpp#L46-L60 since we initialize everything in `SetUp()` anyways. Not gonna fight for it though.

Reply:
Your call.

I I think I'd prefer the long form, but also won't fight for it.
I am afraid you bring U2 to the fight.

Reply:
Maybe we just do a poll in the next roundtable?

Nah, for fighting I use the hurting stuff. Maybe I'll punish you with Schlager, Eurodance, or whoever German rapper is currently in the charts. 🤷‍♀️